src/azure-cli/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -522,9 +522,13 @@ def _build_storage_profile():
             }
         }
 
+    vm_properties['additionalCapabilities'] = {}
     if ultra_ssd_enabled is not None:
         vm_properties['additionalCapabilities'] = {'ultraSSDEnabled': ultra_ssd_enabled}
```

Comment:
> vm_properties['additionalCapabilities'] = {}

Because you declared this object, please also modify this line
```suggestion
        vm_properties['additionalCapabilities']['ultraSSDEnabled'] = ultra_ssd_enabled
```